public/font/Rouge Script/OFL.txt

```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2011 Typesenses (typesenses@live.com.ar),

```

Comment:
Don´t add files that you won´t need to your repository.
I suggest you put this file in your gitignore